x-pack/plugins/security_solution/scripts/run_cypress/print_run.ts

```diff
@@ -277,7 +277,11 @@ function formatFooterSummary(results: CypressCommandLine.CypressRunResult) {
 export function renderSummaryTable(results: CypressCommandLine.CypressRunResult[]) {
   const parsedResults = _.reduce(
     results,
-    (acc: CypressCommandLine.CypressRunResult, result) => {
+    (acc: CypressCommandLine.CypressRunResult, result, index) => {
+      if (!result) {
+        console.warn(`Result for index: ${index} is empty (${result}). Skipping...`);
```

Comment:
Do we also want to add an early return here? Wouldn't it still cause `TypeError: Cannot read properties of undefined (reading 'startedTestsAt')` to be shown?

Reply:
oh yes, of course, that would have been the whole point :D Thanks!